high_card.rb

```diff
@@ -1,5 +1,29 @@
 # Use these two arrays to generate a deck of cards.
+require "pry"
+
+
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+deck = ranks.product(suits).shuffle
```

Comment:
Clever! To me this isn't super clear what it does without looking up the Array#product method, so I'd suggest a clarifying comment here.